app/controllers/submit.js

```diff
@@ -359,6 +415,74 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
+    // Language about submission and moderation.
+    ////////////////////////////////////////////////////
+
+    moderationType: Ember.computed('theme.provider.reviewsWorkflow', function() {
+        return this.get('theme.provider.reviewsWorkflow') ?
+            this.get('theme.provider.reviewsWorkflow').toLowerCase() :
+            null;
+    }),
+    workflow: Ember.computed('moderationType', function () {
+        return WORKFLOW[this.get('moderationType')];
+    }),
+    providerName: Ember.computed('theme.isProvider', function() {
+        return this.get('theme.isProvider') ?
+            this.get('theme.provider.name') :
+            this.get('i18n').t('global.brand_name');
+    }),
+    modalTitle: Ember.computed('moderationType', function() {
+        if (this.get('editMode')) {
+            return MODAL_TITLE['resubmit'];
+        }
+        return this.get('moderationType') === PRE_MODERATION ?
+            MODAL_TITLE['submit'] :
+            MODAL_TITLE['create'];
+    }),
+
+    // submission
+    heading: Ember.computed('moderationType', function() {
+        return this.get('moderationType') === PRE_MODERATION ?
+            ACTION['submit']['heading'] :
+            ACTION['create']['heading'];
+    }),
+    buttonLabel: Ember.computed('moderationType', function() {
+        return this.get('moderationType') === PRE_MODERATION ?
+            ACTION['submit']['button'] :
+            ACTION['create']['button'];
+    }),
+    generalInformation: Ember.computed('moderationType', function() {
+        return this.get('moderationType') ?
+            SUBMIT_MESSAGES['moderation'] :
+            SUBMIT_MESSAGES['default'];
+    }),
+    permissionInformation: Ember.computed('moderationType', function() {
+        return this.get('moderationType') === PRE_MODERATION ?
+            PERMISSION_MESSAGES['submit'] :
+            PERMISSION_MESSAGES['create'];
+    }),
+    moderationInformation: Ember.computed('moderationType', function() {
+        return SUBMIT_MESSAGES[this.get('moderationType')];
+    }),
+
+    // edit
+    showInformation: Ember.computed('moderationType', 'model.reviewsState', function() {
+        let state = this.get('model.reviewsState');
+        let modType = this.get('moderationType');
+        return !(state === ACCEPTED || (modType === POST_MODERATION && state === PENDING));
+    }),
+    editInformation1: Ember.computed('moderationType', function() {
+        return EDIT_MESSAGES['line1'][this.get('moderationType')];
+    }),
+    editInformation2: Ember.computed('moderationType', 'model.reviewsState', function() {
+        return EDIT_MESSAGES['line2'][this.get('model.reviewsState')][this.get('moderationType')];
+    }),
+    canResubmit: Ember.computed('moderationType', 'model.reviewsState', function() {
+        let state = this.get('model.reviewsState');
+        return this.get('moderationType') === PRE_MODERATION && (state === PENDING || state === REJECTED);
```

Comment:
They can't. Since the preprint is already public and may never be approved, people shouldn't be hesitant to make changes (that was my reasoning at least).